src/modules/playRoom/components/CountDown.js

```diff
@@ -1,11 +1,35 @@
+import { useState, useEffect } from 'react';
+import { socket } from '../../../core/services/socket';
+
 const CountDown = () => {
+  const [count, setCount] = useState(60);
+  const [isProcessingResult, setIsProcessingResult] = useState(false);
+  
+  useEffect(() => {
+    socket.on('tick', (timeLeft) => {
+      setCount(timeLeft);
+    });
+    socket.on('processing-result', () => {
+      setIsProcessingResult(true);
+    });
+    socket.on('finish-result', () => {
+      setIsProcessingResult(false);
+    });
+  }, []);
+  
   return (
     <div className="countdown">
-      <p className="countdown-time">
-        Thời gian còn lại để đặt cược: 
-        <br/>
-        <strong> 59 s </strong>
-      </p>
+      {
+        isProcessingResult ? 
+        <p className="countdown-time">
+          Đang xử lý kết quả...
+        </p> :
```

Comment:
why ```:``` is placed in this line?

Reply:
it's conditional rendering syntax

Reply:
Place it in front of false condition, it help your code more clearly

Reply:
In my opinion, the way I wrote is more clearly. It separates 2 conditions and it's the way Shit taught.

Reply:
Well I don't write this code so when I read this code I though ```:``` will be render to UI but not part of condition expression

Reply:
it's okay. Im gonna rewrite those codes follow instruction of react doc